cogs/weather.py

```diff
@@ -0,0 +1,51 @@
+import discord
+from discord.ext import commands
+
+class Roll(commands.Cog):
```

Comment:
The class name here should be something that describes the command, or collection of commands. 
In this case, probably something like "Weather"? 

At the very least, let's not overwrite the Roll command. 😄 
```suggestion
class Weather(commands.Cog):
```